src/Provider/GoogleMapsPlaces/GoogleMapsPlaces.php

```diff
@@ -348,7 +348,7 @@ private function fetchUrl(string $url, array $query): AddressCollection
                 $address = $address->withWebsite($result->website);
             }
 
-            if (isset($result->opening_hours)) {
+            if (isset($result->opening_hours->open_now)) {
```

Comment:
Could we make this change in the model please (ie default to false if no hours)?
https://github.com/geocoder-php/Geocoder/blob/master/src/Provider/GoogleMapsPlaces/Model/OpeningHours.php#L76

Thanks!

Reply:
But wouldn't this  result in a false assumption? Then the user would assume that the place is closed, but in reality we can not make an assertion. The place could be closed or open. Also setting `open_now` to `null` would probably throw off most users, because they would not check if the value is a boolean.

For me personally an empty object or null should be treated the same, as both values imply that no information about the opening hours exist.

Reply:
Fair point. 
Is there any cases where `open_now` isn't set but the opening hours are?

Reply:
Hmm, it seems to be a bit more complicated.

According to the docs, the Text Search Request and the Nearby Search Request only return the field `open_now`. There are no fields `periods` and `weekday_text` for these requests. I could also not find any results that did include any fields other than `open_now`.

https://developers.google.com/places/web-service/search#PlaceSearchResults

The Find Place Request on the other hand might also return a field `weekday_text`. It is only show in the JSON Response, but they did not specify what is returned in the docs. I could not find any information about a field of `periods`, so I don't know where that comes from.

https://developers.google.com/places/web-service/search#find-place-responses

So I don't know if there are cases where `open_now` is not set, but `weekday_text` exists.

Should we add a `hasOpenNowInformation` method to `OpeningHours` and allow `isOpenNow` to be nullable? What would you suggest?

Reply:
I would just make it nullable, and add a note to the docblock mentioning that it will return null if there's no information. As long as we add the correct docblocks, it should show the return types etc in IDEs.